CHANGELOG.md

```diff
@@ -19,9 +19,12 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ### Changed
 
+- [breaking-change] use `fugit::Duraction` for `CountDown::Time` [#381]
```

Comment:
Typo: Duraction -> Duration (also in the PR title). Maybe add "instead of a frequency" to the line?

Reply:
Thanks